lib/ASRouterTargeting.jsm

```diff
@@ -215,4 +220,6 @@ this.ASRouterTargeting = {
   }
 };
 
-this.EXPORTED_SYMBOLS = ["ASRouterTargeting", "removeRandomItemFromArray"];
```

Comment:
`removeRandomItemFromArray` was being exported but then not used anywhere, so I removed it